components/x-live-blog-wrapper/src/LiveBlogWrapper.jsx

```diff
@@ -6,26 +6,31 @@ import { listenToLiveBlogEvents } from './LiveEventListener';
 const withLiveBlogWrapperActions = withActions({
 	insertPost (post) {
 		return ({ posts }) => {
-			const updatedPosts = [ post, ...posts ];
+			posts.unshift(post)
 
-			return { posts: updatedPosts };
+			return { posts };
```

Comment:
Rather than creating and returning a new props object at the end of each of these action methods, I'm wondering if this might be a little clearer:

```javascript
return (props) => {
	props.posts.unshift(post);

	return props;
};
```

My thinking is that this makes it explicit that the argument to this method is a props object and that we're mutating the posts in place.

What do you think?